packages/select/src/Select/Select.tsx

```diff
@@ -615,29 +613,13 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
               </ListMenu>
             </MenuButton>
           </SelectContext.Provider>
-          {state === State.Error && errorMessage && (
-            <span
-              data-lgid={LGIDS_SELECT.errorMessage}
-              className={cx(
-                errorTextStyle({ darkMode, sizeSet }),
-                css`
-                  ${MobileMediaQuery} {
-                    font-size: ${mobileSizeSet.description.text}px;
-                    line-height: ${mobileSizeSet.description.lineHeight}px;
-                  }
-                `,
-                {
-                  [css`
-                    // Hide error text when menu is open,
-                    // so it doesn't peek around the menu corner
-                    color: transparent;
-                  `]: open,
-                },
-              )}
-            >
-              {errorMessage}
-            </span>
-          )}
+          <StateFeedback
```

Comment:
should this component live in FormField as well? (Does it already, should we just export it?)